packages/google_maps_flutter/google_maps_flutter_android/android/src/main/java/io/flutter/plugins/googlemaps/MarkerController.java

```diff
@@ -143,6 +143,9 @@ public void setZIndex(float zIndex) {
     marker.setZIndex(zIndex);
   }
 
+  @Override
+  public void setCollisionBehavior(int collisionBehavior) {}
```

Comment:
Please add a javadoc that describes the valid collisionBehavior int's or at least where they are defined. 
Better yet convert int to something typesafe